desktop/src/gui/dialogs/preferences_dialog.rs

```diff
@@ -261,6 +269,23 @@ impl PreferencesDialog {
         ui.end_row();
     }
 
+    fn show_video_preferences(&mut self, locale: &LanguageIdentifier, ui: &mut Ui) {
+        ui.label(text(locale, "enable-openh264"));
+
+        let previous = self.enable_openh264;
+        ui.checkbox(&mut self.enable_openh264, text(locale, "enable-openh264"));
```

Comment:
Hmm, this looks a bit weird with the label on the left and a checkbox on the right with the same label.

How about `ui.add(Checkbox::without_text(&mut self.enable_openh264))`?

Reply:
Good point, thank you for the suggestion, changed.